pkg/kubelet/network/plugins.go

```diff
@@ -135,7 +140,27 @@ func UnescapePluginName(in string) string {
 type noopNetworkPlugin struct {
 }
 
+const sysctlBridgeCallIptables = "net/bridge/bridge-nf-call-iptables"
+
 func (plugin *noopNetworkPlugin) Init(host Host) error {
+	// Set bridge-nf-call-iptables=1 to maintain compatibility with older
+	// kubernetes versions to ensure the iptables-based kube proxy functions
+	// correctly.  Other plugins are responsible for setting this correctly
+	// depending on whether or not they connect containers to Linux bridges
+	// or use some other mechanism (ie, SDN vswitch).
+
+	// Ensure the netfilter module is loaded on kernel >= 3.18; previously
+	// it was built-in.
+	if _, err := utilexec.New().Command("modprobe", "br-netfilter").CombinedOutput(); err != nil {
+		glog.V(3).Infof("Module br-netfilter not loaded")
+	} else {
+		glog.V(3).Infof("Module br-netfilter loaded")
```

Comment:
This is actually not what I meant. I meant if someone has a CNI plugin that needs br-netfilter, and it currently works because kube-proxy is invoking the load, it will not work after this change. I was asking for the output of modinfo in kube-proxy as a clue that it isn't loaded. This logging statement won't get invoked in the case I'm talking about because we'll init CNI.